OrbitGl/Batcher.h

```diff
@@ -142,9 +149,7 @@ class Batcher {
 
   BatcherId batcher_id_;
   PickingManager* picking_manager_;
-  LineBuffer line_buffer_;
-  BoxBuffer box_buffer_;
-  TriangleBuffer triangle_buffer_;
+  std::unordered_map<float, Buffers> buffers_by_layer_;
```

Comment:
We use absl::flat_hash_map as the default type for maps (unless you have good reasons not to, say performance metrics that prove you have significantly better performance using std::unordered_map).

Reply:
We have an issue with absl::flat_hash_map because we don't have a move constructor in Blockchain. So, either we have to implement a move constructor there or we have to use unordered_map. For now, I will leave the unordered_map.